waku/v2/protocol/waku_rln_relay/waku_rln_relay_utils.nim

```diff
@@ -1220,9 +1244,10 @@ proc mount(node: WakuNode,
       let rlnRelayCredPath = joinPath(conf.rlnRelayCredPath, RlnCredentialsFilename)
       debug "rln-relay credential path", rlnRelayCredPath
       # check if there is an rln-relay credential file in the supplied path
-      if fileExists(rlnRelayCredPath): 
+      if fileExists(rlnRelayCredPath):
+        info "A RLN credential file exists in provided path"
         # retrieve rln-relay credential
-        credentials = some(readPersistentRlnCredentials(rlnRelayCredPath))
+        credentials = readRlnCredentials(rlnRelayCredPath, conf.rlnRelayCredentialsPassword)
```

Comment:
Here you are accessing a `conf` field. Using the `conf` object everywhere makes refactoring/decoupling work harder.

Can you specify a parameter in this function named `rlnRelayCredentialsPassword` and use it here?

Reply:
The advantage is that you decouple the `mount()` procedure from the `WakuConf` type (char2 or wakunode2 conf type), which will ease the task of decoupling the RLN module from the chat2 and wakunode2 configuration types.